data/digests/34873b4d2390.json

```diff
@@ -0,0 +1,110 @@
+                "width": 1500,
+                "height": 1303
+            }
+        }
+    },
+    "subjects": [
+        {
+            "id": "biochemistry-chemical-biology",
+            "name": "Biochemistry and Chemical Biology"
+        }
+    ],
+    "content": [
+        {
+            "type": "image",
+            "image": {
+                "uri": "https://iiif.elifesciences.org/lax:35800/elife-35800-fig4-v1.tif",
+                "alt": "",
+                "attribution": [
+                    "Clip taken from Video 2 of <a href=\"https://doi.org/10.7554/eLife.35800\">Warren et al.</a> (<a href=\"https://creativecommons.org/licenses/by/4.0/\">CC-BY 4.0</a>)"
+                ],
+                "source": {
+                    "mediaType": "image/jpeg",
+                    "uri": "https://iiif.elifesciences.org/lax:35800/elife-35800-fig4-v1.tif/full/full/0/default.jpg",
+                    "filename": "elife-35800-fig4-v1.jpg"
+                },
+                "size": {
+                    "width": 1500,
+                    "height": 1303
+                }
+            },
+            "title": "Images corrected using Galene (right) show much more detail than uncorrected images (left)."
+        },
+        {
+            "type": "paragraph",
+            "text": "High resolution microscopy makes it possible to image events happening in single cells, or even specific parts of a cell. However, small movements like those due to the heartbeat or breathing can blur the images, making it difficult to study living animals. This is particularly problematic for images that take several minutes to capture."
+        },
+        {
+            "type": "paragraph",
+            "text": "Warren et al. have now developed a new open source software tool called Galene. The tool can correct for small movements in images collected by a technique called fluorescence lifetime imaging microscopy (FLIM). As a result, clear images can be captured in situations that were not previously possible. For example, Warren et al. watched cancer cells migrating to the liver of a mouse from the spleen over 24 hours, and, using a fluorescent biosensor, showed that a repurposed drug interferes with how well the cells can attach to the liver. In addition, Warren et al. used the software to take steady 3D images of human skin in a volunteer’s arm, which could be used to study drug penetration."
+        },
+        {
+            "type": "paragraph",
+            "text": "Galene could help researchers to study a wide range of biological processes in living animals. The software can also be applied to existing data to clean up blurred images. In the future Galene could be further developed to work with the imaging techniques used during surgery. For example, surgeons could use it to help them find the edges of tumours."
+        }
+    ],
+    "relatedContent": [
+        {
+            "statusDate": "2018-07-09T00:00:00Z",
+            "id": "35800",
```

Comment:
We should use articles that exist here. Either add this article (which is work), or just reuse an existing one (I did this for the other digests, it's just demo content).